test/quantization/test_quantize_script.py

```diff
@@ -2623,3 +2630,396 @@ def forward(self, x):
         x = torch.rand(5, 5)
         for tracing in [True, False]:
             model = self.checkGraphModeOp(M(), x, "quantized::linear_dynamic", tracing=tracing, dynamic=True)
+
+class TestQuantizeScript(JitTestCase):
```

Comment:
please request review from quantization team next time, this is not the correct place for this test and some of the test cases needs to split further